ext/pdo_oci/tests/oci_success_with_info.phpt

```diff
@@ -0,0 +1,56 @@
+--TEST--
+Handling OCI_SUCCESS_WITH_INFO
+--SKIPIF--
+<?php
+if (!extension_loaded('pdo') || !extension_loaded('pdo_oci')) die('skip not loaded');
+?>
+--FILE--
+<?php
+require __DIR__ . '/../../pdo/tests/pdo_test.inc';
+
+$conn = PDOTest::test_factory('ext/pdo_oci/tests/common.phpt');
+
+$conn->exec(<<<'SQL'
+BEGIN
+    EXECUTE IMMEDIATE 'DROP PROFILE TEST_PROFILE CASCADE';
+EXCEPTION
+    WHEN OTHERS THEN
+        IF SQLCODE != -2380 THEN
+            RAISE;
+        END IF;
+END;
+SQL
+);
+
+$conn->exec(<<<'SQL'
+BEGIN
+    EXECUTE IMMEDIATE 'DROP USER BOB';
+EXCEPTION
+    WHEN OTHERS THEN
+        IF SQLCODE != -1918 THEN
+            RAISE;
+        END IF;
+END;
+SQL
+);
+
+$conn->exec('CREATE PROFILE TEST_PROFILE LIMIT PASSWORD_LIFE_TIME 1/86400 PASSWORD_GRACE_TIME 1');
+$conn->exec('CREATE USER BOB IDENTIFIED BY Passw0rd PROFILE TEST_PROFILE');
+$conn->exec('GRANT CREATE SESSION TO BOB');
+
+// let the password expire
+sleep(2);
+
+$conn = new PDO(getenv('PDOTEST_DSN'), 'BOB', 'Passw0rd');
+var_dump($conn->errorInfo());
+?>
+--EXPECTF--
+array(3) {
+  [0]=>
+  string(5) "HY000"
+  [1]=>
+  int(28002)
+  [2]=>
+  string(%d) "OCISessionBegin: OCI_SUCCESS_WITH_INFO: ORA-28002: the password will expire within %d days
```

Comment:
Use a %s for the English text so that the test passes when messages are not in English.

Reply:
Done. But I'd say it makes more sense to enforce the English locale when connecting to the DB.